students/johnpharmd/lesson03/strformat_lab.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+# started with form_st = 'file_{:d}: {:f}, {:d}, {:f}'
+# Internet search pointed me to https://docs.python.org/2/library/string.html,
+# which, apparently, is not (completely) deprecated in Python 3.
+# Also used https://docs.python.org/3.3/library/string.html#formatstrings.
+# iteration: form_st = 'file_{:03d}: {:f}, {:e}, {:e}'
+# iteration: form_st = 'file_{:03d}: {:.2f}, {:e}, {:e}'
+# iteration: form_st = 'file_{:03d}: {:.2f}, {:.2e}, {:e}'
+# Task 1
+ form_st = 'file_{:03d}: {:.2f}, {:.2e}, {:.2e}'
```

Comment:
Remove the single space indentation through this section. This will throw an IndentationError and cause the program to crash.

Your string formatting operators look good but you need to call the format method on these strings as you did in the functions below. Adding a print statement is a nice way to test that your code works as expected:

```
numbers = (2, 123.4567, 10000, 12345.67)
form_st = 'file_{:03d}: {:.2f}, {:.2e}, {:.2e}'.format(*numbers)
print(form_st)
```